x-pack/plugins/infra/public/components/asset_details/content/content.tsx

```diff
@@ -35,6 +35,16 @@ export const Content = ({
       <TabPanel activeWhen={FlyoutTabIds.ANOMALIES}>
         <Anomalies nodeName={node.name} onClose={overrides?.anomalies?.onClose} />
       </TabPanel>
+      <TabPanel activeWhen={FlyoutTabIds.OVERVIEW}>
+        <Overview
+          currentTimeRange={currentTimeRange}
+          nodeName={node.name}
+          nodeType={nodeType}
+          onTabsStateChange={(tabId: TabIds) => onChange({ activeTabId: tabId })}
```

Comment:
Ok, I added the suggestion and used showTab here instead, but we are not getting rid of the `onChange` function because it's used to handle the other state updates. I added [this commit](https://github.com/elastic/kibana/pull/160924/commits/4a12e3e4b8e563ac0230595c418f699b039c7d55) to apply the changes, I am not sure if I understood the idea correctly so we can discuss the changes there to make sure that it's correct.